### PR TITLE
coq-fourcolor.1.2.2 compatibility release

### PR DESCRIPTION
**Pull request by chdoc**
This release is compatible with `mathcomp-1.11`.

### CODE REVIEW

File: released/packages/coq-fourcolor/coq-fourcolor.1.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+homepage: "https://math-comp.github.io/math-comp/"
```
Comment (palmskog) [inline released/packages/coq-fourcolor/coq-fourcolor.1.2.2/opam:3]:
```suggestion
homepage: "https://math-comp.github.io"
```

File: released/packages/coq-fourcolor/coq-fourcolor.1.2.2/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+bug-reports: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+dev-repo: "git+https://github.com/math-comp/fourcolor"
+license: "CeCILL-B"
```
Comment (palmskog) [inline released/packages/coq-fourcolor/coq-fourcolor.1.2.2/opam:6]:
```suggestion
license: "CECILL-B"
```
The exact SPDX identifier.

**Comment by chdoc** [comment]
I just copied the `opam` file from the previous release and adapted versions and the hash...

**Comment by palmskog** [comment]
Looks good to me now. In an ideal world we could have complete consistency between what is currently in the archive and reviewing practices, but this is not currently feasible.

**Pull request merged**